lib/serialDevice.js

```diff
@@ -371,13 +373,19 @@ module.exports = function(config) {
       }
       if (info.error) {
         debug('Serial send returned ' + info.error);
+        callback(new Error(info.error),null);
       }
-      callback(info.error, info);
+      callback(null, info);
     };
     if (bytes == null) {
       debug('just tried to send nothing!');
     } else {
-      chrome.serial.send(connection.connectionId, bytes, sendcheck);
+      if(connection == null) {
+        debug('Device not connected.');
```

Comment:
I see this logged when I'm busy playing YouTube in the foreground with the uploader in the background, device still very much connected.